modules/service-autoscaling/variables.tf

```diff
@@ -0,0 +1,65 @@
+variable "cluster_name" {
+  description = "ECS cluster name. Uses default cluster if not supplied."
+  default     = ""
+}
+
+variable "service_name" {
+  description = "ECS service name"
+}
+
+variable "max_capacity" {
+  description = "Maximum number of tasks that the autoscaling policy can set for the service."
```

Comment:
This should be set to a reasonable default

Reply:
The only reasonable default is `1`, in my opinion. I'll set it to that.

Reply:
Done

Reply:
That makes sense. The idea being easy as possible to implement and get running with lots of configurations. README should point out that the default is 1, which isn't really useful.